### ISSUE TITLE
How to disable options if already selected

### ISSUE THREAD

**Issue by parwinderD4D** [reactjs/react-autocomplete#391]
Hi Team,,

I am using this package and i want to disable all those option which are already selected....
I have two array 1st contain list of all countries and 2nd one contain list of those country which are 
 selected by user. 

Now i have to disable all those counties which are in 2nd array..

could you please help to fix this problem....

Thanks

**Comment by alexyapp** [comment]
Have you tried using the shouldItemRender prop?

**Comment by parwinderD4D** [comment]
No, could you please send me example code of this.... how i can use it..

Thanks
